dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Goals/GoalRecommendation.cshtml

```diff
@@ -0,0 +1,84 @@
+                @Html.LabelFor(model => model.AssociatedGoal.StartDate, htmlAttributes: new { @class = "control-label" })
+                @Html.EditorFor(model => model.AssociatedGoal.StartDate, new { htmlAttributes = new { @class = "form-control" } })
+            </div>
+            <div class="col-md-3 slim-col">
+                @Html.LabelFor(model => model.AssociatedGoal.EndDate, htmlAttributes: new { @class = "control-label" })
+                @Html.EditorFor(model => model.AssociatedGoal.EndDate, new { htmlAttributes = new { @class = "form-control" } })
+            </div>
+            <div class="col-md-3 slim-col">
+                @Html.LabelFor(model => model.ExpirationDate, htmlAttributes: new { @class = "control-label" })
+                @Html.EditorFor(model => model.ExpirationDate, new { htmlAttributes = new { @class = "form-control" } })
+            </div>
+        </div>
+        <div class="form-group">
+            <div class="col-md-6 slim-col">
+                @Html.LabelFor(model => model.AssociatedGoal.GoalType, htmlAttributes: new { @class = "control-label" })
+                @Html.DropDownListFor(model => model.AssociatedGoal.GoalType, SelectListHelper.FromEnum<GoalType>(Model.AssociatedGoal.GoalType), htmlAttributes: new { @class = "form-control" })
+            </div>
+        </div>
+        <div class="form-group">
+                <div class="col-md-9 slim-col">
+                    @Html.LabelFor(model => model.AssociatedGoal.Range, htmlAttributes: new { @class = "control-label" })
+                </div>
+        </div>
+        <div class="form-group">
+            @Html.EditorFor(model => model.AssociatedGoal.Range, new { htmlAttributes = new { @class = "form-control" } })
+        </div>
+
+        <div class="form-group">
+            <div class="col-md-3 slim-col">
+                @Html.LabelFor(model => model.AssociatedGoal.RecurrenceMetrics.OccurrenceCount, htmlAttributes: new { @class = "control-label" })
+                @Html.EditorFor(model => model.AssociatedGoal.RecurrenceMetrics.OccurrenceCount, new { htmlAttributes = new { @class = "form-control" } })
+            </div>
+            <div class="col-md-3 slim-col">
+                @Html.LabelFor(model => model.AssociatedGoal.RecurrenceMetrics.WindowType, htmlAttributes: new { @class = "control-label" })
+                @Html.DropDownListFor(model => model.AssociatedGoal.RecurrenceMetrics.WindowType, SelectListHelper.FromEnum<ActionPlanScheduleRecurrenceType>(Model.AssociatedGoal.RecurrenceMetrics?.WindowType), htmlAttributes: new { @class = "form-control" })
+            </div>
+        </div>
+        <div class="form-group">
+            <div class="col-md-12 slim-col">
+                @Html.ActionLink("Cancel", "Index", new { personId = Request.Params["personId"], recordId = Request.Params["recordId"] }, new { @class = "btn btn-default" })
+                <input type="submit" value="Save" class="btn btn-default" />
```

Comment:
This was copied form the goal page. Looking at other pages, the save are btn-default